ls1intum/Apollon
# UI Changes to Control Panel

### Checklist

### Motivation and Context
1. When scrolling between the elements of the switches in the control panel, there is no border on the left side of the middle elements. 
2. When switching between Light and Dark Mode there is the obvious indicator of the change, which is the editor changing its color. However, there is no indication on the Light/Dark switch buttons itself.

### Description
1. By removing the "border-left" property from the switch styles, the left border of the element is normal and the change between switch elements looks cleaner.
2. By clicking one of the Light/Dark switches, the `setThemings()` function is called, which takes a string of the button that was clicked. The function then adds the "selected" style property to the selected button and removes the "selected" style property from the unselected one.

### Steps for Testing
1. Run Apollon locally.
2. Open Apollon in Chrome.
3. Check if the borders transition well on the switches when moving between them.
4. Check if the button is selected when changing between light and dark mode.

### Screenshots
### Before:
![Before](https://github.com/ls1intum/Apollon/assets/40467337/3598c73d-d412-4a89-9dbe-99b157ab791e)


### After:
![After](https://github.com/ls1intum/Apollon/assets/40467337/9191895f-e7cc-471b-9416-6bc4f9035d3c)
